packages/design-system-react-native/src/components/Text/Text.tsx

```diff
@@ -0,0 +1,40 @@
+import { useTailwind } from '@metamask/design-system-twrnc-preset';
+import React from 'react';
+import { Text as RNText } from 'react-native';
+
+import { DEFAULT_TEXT_COLOR, DEFAULT_TEXT_VARIANT } from './Text.constants';
+import type { TextProps } from './Text.types';
+import { FontWeight, FontStyle } from './Text.types';
+import { generateClassNames } from './Text.utilities';
+
+const Text: React.FC<TextProps> = ({
+  variant = DEFAULT_TEXT_VARIANT,
+  color = DEFAULT_TEXT_COLOR,
+  style,
+  children,
+  fontWeight = FontWeight.Normal,
+  fontStyle = FontStyle.Normal,
+  twClassNames = '',
```

Comment:
Now that I can see we can add custom classnames maybe `className` would be an appropriate prop name to improve consistency across platforms. If not `className` I think it may make sense to make the prop name singular `twClassName` remove the `s`

Reply:
`twClassName` works